README.md

```diff
@@ -165,7 +182,7 @@ Example (`make test`, `make fix-fixtures`):
 export FASTLY_TEST_SERVICE_ID="SERVICEID"
 export FASTLY_API_KEY="TESTAPIKEY"
 
-make test TESTARGS="-run=NewClient"
```

Comment:
`NewClient` doesn't exist and through me off for a while when I was trying to test.